storybook/storybook-css/src/docs/focus.docs.mdx

```diff
@@ -0,0 +1,16 @@
+import { Meta } from "@storybook/blocks";
+
+<Meta title="Docs/Designrichtlijnen/Focus staat" />
+
+# Focus staat
+
+De toegankelijkheidsrichtlijnen eisen dat het element op de pagina dat focus heeft visueel herkenbaar is. Dit geldt ook voor formuliervelden. Houd rekening bij het ontwerp en de ontwikkeling van de website of applicatie met de focus staat.
```

Comment:
Here comes a more general point, but this seems a good moment to make it. I think it would be stronger to demand indicating a focused element ourselves, rather than ‘hide behind’ a law or regulation. How about just “Een element dat focus heeft moet visueel herkenbaar zijn. Dit geldt voor interactieve elementen zoals links, knoppen en formuliervelden.”

The last sentence may be unnecessary, as we implement focus state in our components, so that implementers don’t need to think about it.

Reply:
I agree. I got this text from the [current design system page](https://amsterdam.nl/digitalservices/designrichtlijnen/focus-staat/), but a more active voice is preferable.

I think we need to keep that last sentence though, the reason I added this doc now is because our devs weren't aware of how we handle the focus indicator.